.gitignore

```diff
@@ -2,11 +2,14 @@
 /.gradle/
 /build/
 .idea/
-arisa.yml
 helper-messages.json
 /.kotest
 /bin/
 /lib/
 /out/
 /logs/
-last-run
\ No newline at end of file
+last-run
+
+# Old credentials config file
+arisa.yml
+/config/local.yml
```

Comment:
I would put config/local.yml, so it works in windows

Reply:
I am on Windows and this is not an issue for me at least. `.gitignore` paths are relative to the directory in which the `.gitignore` is placed.